### ISSUE TITLE
add OS detection to provide recommended downloads

### ISSUE THREAD

**Issue by jolan** [decred/dcrweb#69]
An older revision of the site had some OS detection logic based upon platform.js:

https://github.com/bestiejs/platform.js/

We should bring this logic back so we can i.e. link Windows users to Paymetheus and macOS/Linux users to the proper version of Decrediton on the main site.

Maybe also do this on the downloads page for the CLI installer/gominer and provide direct links to the correct versions.

Basically we really want to do as much as we can to avoid dumping people onto the GitHub binaries/releases pages which are extremely difficult to navigate.

**Comment by pvtwarren** [comment]
pretty sure #75 Fixed this :)

**Comment by jolan** [comment]
It's only partially implemented. We would like to add it on the downloads page as well.

**Comment by noahpierau** [comment]
Can this be closed? @karamble

**Comment by jolan** [comment]
No, this is still only partially implemented.

**Comment by jolan** [comment]
I was just thinking it should emphasize the detected OS for each download followed by a de-emphasized line linking to the downloads for other operating systems.  i.e.:

#### Download Decrediton For Windows
###### Or Download for Mac, Linux

**Comment by jzbz** [comment]
I think this can be closed. OS detection works and a link to the full downloads section is provided below the recommendation.
